### ISSUE TITLE
Element 0 overwritten?

### ISSUE THREAD

**Issue by jumpjack** [Scopevisio/eudgc#4]
https://github.com/Scopevisio/eudgc/blob/9af2d1baf7c2a802df978d9c195a470a09da1aff/src/cose1.ts#L107

Are you intentionally overwriting element 0 of raw[]?

**Comment by srutz** [comment]
yes, i do that because of the behavior in the library. 0x84 means array of length 4. This way I got it to work. Maybe newer versions of the cbor-web library work without this ugly hack. That would be nice actually. Leaving out this code would break the parsing when I originally wrote the eudgc code.
